src/extension.ts

```diff
@@ -345,7 +349,95 @@ function showUpdateMessage(version: string) {
     });
 }
 
-function copyToClipboard(code: string) {
-    copyPaste.copy(code);
-    window.showInformationMessage("Step Implementation copied to clipboard");
+function putStubImplementationInFileContent(cwd, filePath, stepTexts) {
+    return new Promise((resolve,reject) => {
+        clients.get(cwd).sendRequest("gauge/putStubImpl", {implementationFilePath: filePath, stepTexts: stepTexts}, new CancellationTokenSource().token).then(
+            (val: {fileName: string, fileContent: string}) => {
+                resolve(val.fileContent)
+            },
+            (reason) => {
+                reject(reason)
+            }
+        );
+    })
+}
+
+function writeToFileInTextEditor(filePath, contents) {
```

Comment:
WorkspaceEdits should be applied. We should not open file and write to it explicitly